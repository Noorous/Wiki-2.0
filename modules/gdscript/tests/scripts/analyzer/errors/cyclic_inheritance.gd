class InnerA extends InnerB:
	pass

class InnerB extends InnerA:
	pass

func test():
	pass
