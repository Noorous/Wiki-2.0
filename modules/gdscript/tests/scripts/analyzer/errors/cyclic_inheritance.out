GDTEST_ANALYZER_ERROR
>> ERROR at line 1: Cyclic inheritance.
